packages/kbn-field-types/BUILD.bazel

```diff
@@ -0,0 +1,96 @@
+
+load("@npm//@bazel/typescript:index.bzl", "ts_config", "ts_project")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library", "pkg_npm")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler")
+
+PKG_BASE_NAME = "kbn-field-types"
+PKG_REQUIRE_NAME = "@kbn/field-types"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+  ],
+  exclude = [
+    "**/*.test.*",
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+  "README.md"
+]
+
+RUNTIME_DEPS = [
+  "@npm//jest-styled-components",
+  "@npm//node-forge",
+]
+
+TYPES_DEPS = [
+  "@npm//@types/flot",
+  "@npm//@types/jest",
+  "@npm//@types/node",
+  "@npm//@types/node-forge",
+  "@npm//@types/testing-library__jest-dom",
+  "@npm//resize-observer-polyfill",
+  "@npm//@emotion/react",
+  "@npm//jest-styled-components",
+]
+
+jsts_transpiler(
+   name = "target",
```

Comment:
could we change it to `target_node`? We're trying to make this a standard across packages